test/e2e/e2e_test.go

```diff
@@ -842,6 +842,8 @@ func TestImportExport(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	time.Sleep(time.Second * 5)
```

Comment:
Would be great to figure out why we need this.

Reply:
The `ImportExport` test is now flaky and my theory was that adding a delay between when we start the runtime and import the pod to when we get an inference would help - it, unfortunately, does not. so I'll remove this delay and keep debugging.